cogs/conversioncog.py

```diff
@@ -0,0 +1,27 @@
+from discord.ext import commands
+from unit_converter.converter import convert, converts
+
+
+class ConversionCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot  # This line in the constructor is required
+
+    @commands.command()
+    async def convert(
+        self, ctx, inputunit, outputunit
+    ):  # the ctx parameter is short for 'context'
+        """type your input and output units"""
+        save = convert(inputunit, outputunit)
+
+        await ctx.send(

```

Comment:
This would be nice as an embed. Try the following after adding `import discord` to the top of your file

```python
embed = discord.Embed(title="Unit Conversion")
embed.add_field(name="Input Quantity", value=inputunit)
embed.add_field(name="Output Unit", value=outputunit)
embed.add_field(name="Result", value=save)

await ctx.send(embed=embed)
```

Feel free to give it some colour if you'd like :slightly_smiling_face: it is up to you.